### PR TITLE
fix: Add RayyanRNKJ

### LINKED ISSUE #837: Rayyan Rasheed
CapturePage(4)

### PR DESCRIPTION
**Pull request by RayyanRNKJ**
resolve #837

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1582,5 +1582,11 @@
     "githubUsername": "ChoudharyTara",
     "imageUrl": "https://user-images.githubusercontent.com/87922322/141666358-ffc20544-4424-4e6e-a702-9e0912082b44.png",
     "issueNumber": 835
-}
+  },
+  {
+  "name": "Rayyan Rasheed",
+  "githubUsername": "RayyanRNKJ",
+  "imageUrl": "https://user-images.githubusercontent.com/64679235/141696204-3084d6cf-49d0-452a-979e-35a58b89b4dc.png",
+  "issueNumber": 837
```
Comment (Panquesito7) [inline data.json:1590]:
```suggestion
      "name": "Rayyan Rasheed",
      "githubUsername": "RayyanRNKJ",
      "imageUrl": "https://user-images.githubusercontent.com/64679235/141696204-3084d6cf-49d0-452a-979e-35a58b89b4dc.png",
      "issueNumber": 837
```

File: data.json [hunk 2]
```diff
@@ -1582,5 +1582,11 @@
     "githubUsername": "ChoudharyTara",
     "imageUrl": "https://user-images.githubusercontent.com/87922322/141666358-ffc20544-4424-4e6e-a702-9e0912082b44.png",
     "issueNumber": 835
-}
+  },
```
Comment (Panquesito7) [inline data.json:1585]:
```suggestion
},
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please see https://github.com/EddieHubCommunity/awesome-github-profiles/pull/838/files#r748902485. 😄

File: data.json [hunk 3]
```diff
@@ -1582,5 +1582,11 @@
     "githubUsername": "ChoudharyTara",
     "imageUrl": "https://user-images.githubusercontent.com/87922322/141666358-ffc20544-4424-4e6e-a702-9e0912082b44.png",
     "issueNumber": 835
-}
+},
+  {
```
Comment (schmelto) [inline data.json:1586]:
```suggestion
  },
  {
```

**Pull request closed without merging**